scripts/genproto.sh

```diff
@@ -10,13 +10,13 @@ if ! [[ "$0" =~ scripts/genproto.sh ]]; then
 	exit 255
 fi
 
-if [[ $(protoc --version | cut -f2 -d' ') != "3.7.1" ]]; then
-	echo "could not find protoc 3.7.1, is it installed + in PATH?"
-	exit 255
-fi
+#if [[ $(protoc --version | cut -f2 -d' ') != "3.7.1" ]]; then
```

Comment:
Re-enable or add comment explaining why we don't need this any more?

Reply:
Re-enabled. Good catch.

Would be happy to add a comment why we need this. 
Lookled over changes 3.7..3.12, and there seems nothing go related:  https://github.com/protocolbuffers/protobuf/releases?before=v3.7.1. 

I assume its to guarantee better repeatability for patch releases.